scripts/aspects_migration.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+// utility for reading in the entire curriculum from a file path
+const fs = require('fs')
+const Insight = require('../lib/insight')
+const childProcess = require('child_process').execSync
+
+let curriculumPath
+let topic
+let curriculum
+
+const aspects = ['new', 'workout', 'deep', 'introduction', 'obscura']
+
+function loadCurriculumFromLocalPath () {
+  process.argv.forEach((val, index) => {
+    if (index === 2) {
+      console.log(`${index}: ${val}`)
+      curriculumPath = val
+    } else if (index === 3) {
+      console.log(`${index}:${val}`)
+      topic = val
+    }
+  })
+  curriculum = childProcess(`find ${curriculumPath}/${topic} -name "*.md"`).toString('utf-8')
```

Comment:
instead of building the path yourself, look into `path` module, especially the `.join` method. this will ensure it'll work on any OS, not only unix

Reply:
also not sure you need the argument on the `.toString()`

Reply:
I think running this wil also get the the filepath of the the courses and workouts as they match the regex you passed to `find`

Instead you could do something like:
```
execSync(`find ${curriculumPath/topicPath} -name "*.md"`)
  .toString()
  .split('\n')
  .filter(filePath => !filePath.endsWith('README.md'))
  // this will remove from you array all path ending with README.md which will be Courses and Workouts manifests
```

Reply:
as a mention, you can also pass to `execSync` a  second argument `{cwd: path}`  which will run the command as if you ran it within the the directory youself.
however, this will get you the relative paths and not the absolute ones so at the end you'd need to append a:
```
exeSync('...')
  . ...
  .map(relativePath => path.join(topicPath, relativePath)
```
not really necesarry, but fyi